Адаптивная графика

Roman3341 opened the pull request.

Roman3341 committed:
Оптимизированные изображения

Скачал изображения. Оптимизировал

Roman3341 committed:
Подключение контентных изображений

Подключил контентные изображения в разметку на 3х страницах. При необходимости дополнял разметку. Карту не подключал

Komnatnaya commented:
![image](https://github.com/user-attachments/assets/a723cf44-f382-4e27-b59e-2de217a09ff7)
два заголовка h1 на странице

Komnatnaya commented:
На странице не подключена картинка карты

Roman3341 committed:
Исправление замечайни №1

ИСправил замечание по заголовку. Подключил изображение карты. На странице програм подключил изображения в подвале и хэдере
